common-arcana.lic

```diff
@@ -92,22 +92,31 @@ module DRCA
     end
   end
 
-  def prepare?(abbrev, mana, symbiosis = false, command = 'prepare', tattoo_tm = false)
+  def prepare?(abbrev, mana, symbiosis = false, command = 'prepare', tattoo_tm = false, runestone_name = false, runestone_tm = false)
     return false unless abbrev
 
     DRC.bput('prep symb', 'You recall the exact details of the', 'But you\'ve already prepared') if symbiosis
 
-    match = DRC.bput("#{command} #{abbrev} #{mana}", get_data('spells').prep_messages)
+    if runestone_name
+      match = DRC.bput("#{command} my #{runestone_name}", get_data('spells').invoke_messages)
+    else
+      match = DRC.bput("#{command} #{abbrev} #{mana}", get_data('spells').prep_messages)
+    end
     case match
     when 'Your desire to prepare this offensive spell suddenly slips away'
       pause 1
       return prepare?(abbrev, mana, symbiosis, command, tattoo_tm)
     when 'Something in the area interferes with your spell preparations'
       DRC.bput('rel symb', 'You release the', 'But you haven\'t') if symbiosis
       return false
+    when 'Well, that was fun'
+      DRCI.dispose_trash(runestone_name)
+      return false
+    when 'You\'ll have to hold it'
+      return false
     end
 
-    DRC.bput("target", get_data('spells').prep_messages) if tattoo_tm
+    DRC.bput("target", get_data('spells').prep_messages) if tattoo_tm || runestone_tm
```

Comment:
Do we need to explicitly define `runestone_tm` or would is be sufficient to check that the spell skill is Targeted Magic?

Reply:
I can try checking the skill. What you see here is the results of many, many edits and hours of testing/using.

Reply:
I could leave `runestone_tm` here and pass the skill check to it. Then, I could remove requiring it hard coded to the spell in the yaml. I will test and see if I can get it to work the way it is now.